packages/paste-website/src/pages/patterns/pricing/index.mdx

```diff
@@ -0,0 +1,478 @@
+export const meta = {
+  title: 'Pricing',
+  description: 'The Pricing pattern shows a group of paid features or packaged plans a user is selecting.',
+  slug: '/patterns/pricing/',
+};
+
+import {Anchor} from '@twilio-paste/anchor';
+import {Callout, CalloutHeading, CalloutText} from '@twilio-paste/callout';
+import {UnorderedList, ListItem} from '@twilio-paste/list'
+import {Table, THead, TBody, Tr, Td, Th} from '@twilio-paste/table';
+import {Box} from '@twilio-paste/box';
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+
+import DefaultLayout from '../../../layouts/DefaultLayout';
+import {getFeature, getNavigationData} from '../../../utils/api';
+
+export default DefaultLayout;
+
+export const getStaticProps = async () => {
+  const navigationData = await getNavigationData();
+  const feature = await getFeature('Pricing');
+  return {
+    props: {
+      data: {
+        ...feature,
+      },
+      navigationData,
+    },
+  };
+};
+
+
+<NormalizedPatternHeader description={meta.description} name={meta.title} data={props.data} />
+
+<contentwrapper>
+
+<PageAside data={mdxHeadings} />
+
```

Comment:
**polish:**  Let's add this image above the list of components. 
![Pricing example](https://github.com/user-attachments/assets/d7c2fede-6a7d-4679-a158-0b3806cced54)